distribution/docs/src/main/resources/content/_tables/ContentFileSystemStorageProvider.adoc

```diff
@@ -5,7 +5,7 @@
 :application: ${ddf-catalog}
 :summary: Content File System Storage Provider.
 
-.[[org.codice.ddf.catalog.content.impl.FileSystemStorageProvider]]Content File System Storage Provider
+.[[_org.codice.ddf.catalog.content.impl.FileSystemStorageProvider]]Content File System Storage Provider
```

Comment:
❓ We refer to this link by `{application-prefix}org.codice.ddf.catalog.content.impl.FileSystemStorageProvider` in another place in this PR, should this have the `{application-prefix}`?

Reply:
see above